x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -374,12 +384,52 @@ func TestExecuteContract(t *testing.T) {
 	require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &instantiateResponse))
 
 	specs := map[string]struct {
-		addr   string
-		expErr bool
+		addr      string
+		expEvents []abci.Event
+		expErr    bool
 	}{
-		"address can execute a contract": {
-			addr:   myAddress.String(),
-			expErr: false,
+		"adress can execute a contract": {
+			addr: myAddress.String(),
+			expEvents: []abci.Event{
+				createMsgEvent(myAddress), {
+					Type: "execute",
+					Attributes: []abci.EventAttribute{{
+						Key:   []byte("_contract_address"),
+						Value: []byte("link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8"),
```

Comment:
Please use `instantiateResponse.Address`

Reply:
Thanks. I fixed it as pointed out.